### DISCUSSION TITLE
ORPHA vocabulary

### DISCUSSION THREAD

**Discussion by vmateosresin** [OHDSI/Vocabulary-v5.0#762] [Q&A] [answered]
Hi, 

Do have in mind extend the current list of vocabularies? For instance, in an European project some partners use orpha codification for their concepts, and it not covered by Athena. 

Regards.

**Comment by mik-ohdsi** [comment, marked as answer]
Orpha coding is indeed a possibility, more so as UMLS is now making this available in their delivery.
Please vote this answer up, if you have an immediate need and use case for having Orpha coding as an OHDSI standardized vocabulary in Athena (and provide your use case in a comment, if possible).
regards - mik

**Reply by vmateosresin** [reply]
Hi Mik

Thank you very much for your quick answer. 

In our project, we have several data providers, each one with their own condification strategies, technologies, and so on. In order to standarize all the data formats and codifications, we merge all the data into a common data model, homogenizing the terminology using athena and omop vocabularies and the concept infrastructure. One of the codificacion used by a partner is Orpha, and we are trying to translate the concepts to omop standard concepts, mainly using snomed codification, but it will be great if Orpha is one the included vocabularies. 

Hope this helps to understand the use case!!!

Regards,
Victor

**Comment by TinyRickC137** [comment]
Hi, @vmateosresin, if you are still interested in ORPHA codes, please write me an email at zhuk@ohdsi.org 

There are other scientific groups who are interested in the implementation of these codes in OHDSI vocabularies through the [community contribution](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Community-contribution), we all can benefit from the collaboration.

Best, 
Oleg
